### PR TITLE
DR patterns january

### PR DESCRIPTION
**Pull request by klaudiarybicka**

### CODE REVIEW

File: src/hmrc-content-guide/tariff-stop-press-notices/index.njk [hunk 1]
```diff
@@ -14,6 +14,12 @@ layout: twoColumnPage.njk
 Use this pattern to tell users about changes to the UK Trade Tariff, Customs Handling of Import and Export Freight (CHIEF), Customs Declaration Service (CDS) and monthly trade euro rate.
 </p>
 
+<h2 class="govuk-heading-l">When not to use</h2>
+
+<p class="govuk-body">
+  Don't update a customs information paper after it's been published, unless it's to correct a typo. A new one must be issued.
```
Comment (Jon-Rowe-HMRC) [inline src/hmrc-content-guide/tariff-stop-press-notices/index.njk:20]: Please change 'Don't' to 'Do not'. Negative contractions are advised against in our style guide. We probably have this issue on other pages as just noticed on http://localhost:3000/hmrc-content-guide/customs-information-papers/ too.

File: src/hmrc-content-guide/customs-information-papers/index.njk [hunk 2]
```diff
@@ -52,17 +52,31 @@ Description (CIP #)
...
+Example of a summary: HMRC’s policy for importers who have unintentionally received goods from 1 January 2021 to 31 December 2021 in Great Britain, before they’ve had the opportunity to declare them.
 </p>
 
 <h3 class="govuk-heading-m">Collection page</h3>
 
 <p class="govuk-body">
 Add the document to the <a href="https://www.gov.uk/government/collections/customs-information-papers--2" class="govuk-link">Customs information papers</a> collection page. Do not create a public change note.
+</p>
+
+<h2 class="govuk-heading-l">HTML attachment</h2>
```
Comment (Jon-Rowe-HMRC) [inline src/hmrc-content-guide/customs-information-papers/index.njk:70]: Please make 'HTML attachment' under 'How it works' in the title structure. See http://localhost:3000/hmrc-content-guide/anti-dumping-measures/ as an example.

**Review by Jon-Rowe-HMRC — CHANGES_REQUESTED** [review]
I've made a couple of minor in-line comments. Thanks.

**Comment by klaudiarybicka** [comment]
I have added "Publication" medium header to comply with the new hierarchy in customs information papers, please let me know if that is fine. I also moved the "Collection page" section to the bottom as it is in other patterns.

**Review by Jon-Rowe-HMRC — APPROVED** [review]
Looks good to me, thanks Klaudia

**Pull request merged**